### PR TITLE
Arthur ashe

### PR DESCRIPTION
**Pull request by B77Mills**
Currently the page does the following.

 - Title/Description pulled from management
 - It then loops over its list of child sections(Female Sport Winner, Male Sport Winner, Female Sport Finalists & Male Sport Finalists) displays a list of cards for content scheduled to that child section
 -- Each card should use primaryImage, sidebarName Name & sidebarBody...the sidebar must have the lable of **Scholar Profile**
 - Past Scholars & More About This Scholarship Boxes(can be update to whatever)
 - Latest content block pulling content scheduled to the given section.  

### Landing Page:
aa-award

### 2021 Landing Page
aa2021

### CODE REVIEW

**Review by brandonbk — CHANGES_REQUESTED** [review]
* Male & Female instead of Men & Women
* Singular "Winner"
* Plural "Finalists"
* Add "Sport"...e.g. "Male Sport Winner", "Female Sport Finalists"

File: packages/global/components/nodes/awards.marko [hunk 1]
```diff
@@ -134,13 +134,14 @@ $ const imageLink = primaryImage.src ? { href: content.siteContext.path, attrs:
...
-      <@text
-        class="btn btn-primary btn-block"
-        show=displayButton
-        href=content.linkUrl
-        title=content.linkText
-      >
-          ${callToAction}
+      <@text show=displayButton>
+        <marko-web-link
+            class="btn btn-primary btn-block"
```
Comment (zarathustra323) [inline packages/global/components/nodes/awards.marko:139]: Fix indentation.

File: sites/diverseeducation.com/server/components/nodes/arthur-ashe-profiles.marko [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { get, getAsObject, getAsArray } from "@parameter1/base-cms-object-path";
+import defaultValue from "@parameter1/base-cms-marko-core/utils/default-value";
+
+$ const content = getAsObject(input, "node");
+$ const profile = getAsArray(content, "profiles")[0];
```
Comment (zarathustra323) [inline sites/diverseeducation.com/server/components/nodes/arthur-ashe-profiles.marko:5]: Loop over all profiles.

File: sites/diverseeducation.com/server/templates/website-section/awards-honors/arthur-ashe/index.marko [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+                </default-theme-card-deck-flow>
+              </div>
+            </marko-web-query>
+          </for>
+        </div>
+      </marko-web-query>
+    </@section>
+  </if>
+  <else>
+    <@section modifiers=["arthur-ashe", "arthur-ashe-profiles"]>
```
Comment (zarathustra323) [inline sites/diverseeducation.com/server/templates/website-section/awards-honors/arthur-ashe/index.marko:82]: Componentize the resusable pieces

**Review by zarathustra323 — CHANGES_REQUESTED** [review]

File: sites/diverseeducation.com/server/components/nodes/arthur-ashe-profiles.marko (hunk 2 above)
Comment (B77Mills) [reply sites/diverseeducation.com/server/components/nodes/arthur-ashe-profiles.marko:5]: I left this as it is on the given node and is used for the name and body being displayed on the page

**Pull request merged**